apps/web/src/features/finance/components/detail-sections/payment-basic-info.tsx

```
@@ -0,0 +1,300 @@
+							className="mt-1 h-10"
+						/>
+					) : (
+						<p className="font-mono text-sm">
+							{payment.stripe_transaction_id || "Not provided"}
+						</p>
+					)}
+				</div>
+				<div>
+					<label className="font-medium text-muted-foreground text-sm">
+						Status
+					</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)